sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -670,6 +703,7 @@ export interface ExtractiveQueryAnswer {
     // (undocumented)
     answerType: "extractive";
     count?: number;
+    maxCharLength?: number;
```

Comment:
I don't love this name, but having a hard time coming up with something better. `maxCharacters`? What do you think about that (and feel free to prefer the current name)

Reply:
As per offline, we're going with `maxAnswerLength` and `maxCaptionLength`.